src/main.rs

```diff
@@ -89,7 +86,18 @@ async fn main() {
 
     log::warn!("Starting hithit bot");
 
-    let bot = Bot::from_env().auto_send();
+    let url = std::env::var("BOT_SERVER").unwrap_or_else(|_| {
+        option_env!("BOT_SERVER")
+            .unwrap_or("https://api.telegram.org")
+            .to_string()
+    });
+    let bot = Bot::from_env()
+        .set_api_url(
+            reqwest::Url::parse(&url)
+                .map_err(|e| panic!("Parse telegram bot api url error. {:?}", e))
+                .unwrap(),
```

Comment:
What about `url.parse().expect(...)`? By doing so we avoid depending on reqwest directly.